Packs/Gurucul/ReleaseNotes/1_1_2.md

```diff
@@ -0,0 +1,5 @@
+#### Integrations
+##### Gurucul-GRA
+
+- **Bug Fixes**:
+  - **Breaking Change**: Fixed an issue with fetch incidents.
```

Comment:
@gra-xsoar Why the breaking changes?

Reply:
its incident fetch change that's why we wrote: "Breaking change"?
if not required then we will update the release file